src/assets/styles/components/_banner.scss

```diff
@@ -0,0 +1,57 @@
+.banner {

```

Comment:
remove grid system and use flex
use clip path
height in .banner only 
base color :default
create classes for all basic colors : primary, secondary accent success warning info danger

Reply:
done